### ISSUE TITLE
Error parsing a turtle file with certain language tags

### ISSUE THREAD

**Issue by koenedaele** [rdfhdt/hdt-cpp#58]
When using rdf2hdt on the following turtle file:

```turtle
@prefix dcterms: <http://purl.org/dc/terms/> .
@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
@prefix skos-thes: <http://purl.org/iso25964/skos-thes#> .
@prefix void: <http://rdfs.org/ns/void#> .
@prefix xml: <http://www.w3.org/XML/1998/namespace> .
@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .

<urn:x-skosprovider:trees/3> a skos:Collection ;
    dcterms:identifier 3 ;
    skos:inScheme <urn:x-skosprovider:trees> ;
    skos:member <urn:x-skosprovider:trees/1>,
        <urn:x-skosprovider:trees/2> ;
    skos:prefLabel "Trees by species"@en,
        "Bomen per soort"@nl-BE .

<urn:x-skosprovider:trees/1> a skos:Concept ;
    dcterms:identifier 1 ;
    skos:closeMatch <http://id.python.org/different/types/of/trees/nr/1/the/larch> ;
    skos:definition "A type of tree."@en ;
    skos:inScheme <urn:x-skosprovider:trees> ;
    skos:prefLabel "The Larch"@en,
        "De Lariks"@nl .

<urn:x-skosprovider:trees/2> a skos:Concept ;
    dcterms:identifier 2 ;
    skos:altLabel "la châtaigne"@fr,
        "De Paardekastanje"@nl ;
    skos:definition "A different type of tree."@en ;
    skos:inScheme <urn:x-skosprovider:trees> ;
    skos:prefLabel "The Chestnut"@en ;
    skos:relatedMatch <http://id.python.org/different/types/of/trees/nr/17/the/other/chestnut> .

<urn:x-skosprovider:trees> a skos:ConceptScheme ;
    dcterms:identifier "TREES" ;
    skos:hasTopConcept <urn:x-skosprovider:trees/1>,
        <urn:x-skosprovider:trees/2> ;
    skos:prefLabel "Different types of trees"@en,
        "Verschillende soorten bomen"@nl .
```

I get the following error:

```bash
[21:34:00] $ /home/koen/Projecten/general/hdt-cpp/hdt-lib/tools/rdf2hdt TREES-full.ttl TREES-full.hdt
expected `(', not `B'y: 0 K triples processed.: 0 % / 0 %                      
caught here??
Catch exception load: Error parsing input.
ERROR: Error parsing input.
```

Using an rdfxml version of the same file I have no problems:
```bash
[21:39:04] $ /home/koen/Projecten/general/hdt-cpp/hdt-lib/tools/rdf2hdt -f rdfxml TREES-full.rdf TREES-full.hdt
RDF format: rdfxml
HDT Successfully generated.                                                    
Total processing time: Clock(10 ms 901 us)  User(10 ms 516 us)  System(552 us)
```

Both the Turtle and the rdfxml version were generated from Python using RDFLIB, on Ubuntu 14.04

I believe rdf2hdt use serd for reading turtle files. Using serdi on the same file gives met the following error:
```bash
[21:42:26] 1 $ serdi TREES-full.ttl
<urn:x-skosprovider:trees/3> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://www.w3.org/2004/02/skos/core#Collection> .
<urn:x-skosprovider:trees/3> <http://purl.org/dc/terms/identifier> "3"^^<http://www.w3.org/2001/XMLSchema#integer> .
<urn:x-skosprovider:trees/3> <http://www.w3.org/2004/02/skos/core#inScheme> <urn:x-skosprovider:trees> .
<urn:x-skosprovider:trees/3> <http://www.w3.org/2004/02/skos/core#member> <urn:x-skosprovider:trees/1> .
<urn:x-skosprovider:trees/3> <http://www.w3.org/2004/02/skos/core#member> <urn:x-skosprovider:trees/2> .
<urn:x-skosprovider:trees/3> <http://www.w3.org/2004/02/skos/core#prefLabel> "Trees by species"@en .
<urn:x-skosprovider:trees/3> <http://www.w3.org/2004/02/skos/core#prefLabel> "Bomen per soort"@nl- .
error: TREES-full.ttl:16:29: expected `.', not `B'
```

**Comment by RubenVerborgh** [comment]
This is a SERD error that has been fixed. Make sure you compile with the
latest SERD.

**Comment by koenedaele** [comment]
Thx. Downloaded http://download.drobilla.net/serd-0.26.0.tar.bz2 and after installing the error is fixed.

Do you know what version of serd this issue was fixed in? Ubuntu 16.04 uses libserd 0.22. Would it work out of the box with that version?

**Comment by RubenVerborgh** [comment]
Fixed in https://github.com/drobilla/serd/commit/b1be377295ab70e4462a02f1299d393bd9e07a17, which landed into v0.24.0.
